e2e/tests/notebook/notebook-snippets.spec.ts

```diff
@@ -30,9 +30,8 @@ describe('Notebook Snippets Tests', () => {
       [
         'All of Us Dataset Builder R snippets',
         'All of Us R and Cloud Storage snippets',
-        'All of Us R and SQL snippets'
-        // nope, this one does not exist
-        // 'All of Us Cromwell Setup R snippets'
+        'All of Us R and SQL snippets',
+        'All of Us Cromwell Setup snippets'
```

Comment:
I realized the All of Us Cromwell Setup is not following the naming convention and should be 'All of Us Cromwell Setup R snippets', however on the other hand since it is appearing in R it is redundant to include R in name...thoughts